devops/openshift/database-backup/backup-build.yaml

```diff
@@ -73,4 +71,4 @@ parameters:
     displayName: FROM Image Tag
     description: Base image to build from.  Docker creds or Artificatory setup may be needed to alleviate docker rate-limiting
     required: true
-    value: artifacts.developer.gov.bc.ca/docker-remote/centos/postgresql-12-centos7:20200917-804ef01
+    value: docker.io/centos/postgresql-12-centos7:20200917-804ef01
```

Comment:
Can we claim that current backup supports postgres 14 with this image?

Reply:
Yes it supports Postgres 14, as per the fedora postgres 14 base image